README.md

```diff
@@ -13,70 +13,70 @@
-| AAD              | [azure-spring-boot-starter-active-directory:3.7.0]                                             | [aad-webapp-resource-server](aad/azure-spring-boot-starter-active-directory/aad-web-application-and-resource-server)              |
-| appconfiguration | [azure-spring-boot-starter-cosmos:3.7.0]                                                       | [azure-appconfiguration-conversion-sample-initial](appconfiguration/azure-appconfiguration-conversion-sample-initial)             |
+| AAD              | [azure-spring-boot-starter-active-directory-b2c:3.8.0]                                         | [aad-b2c-resource-server](aad/azure-spring-boot-starter-active-directory-b2c/aad-b2c-resource-server)                             |
+| AAD              | [azure-spring-boot-starter-active-directory-b2c:3.8.0]                                         | [aad-b2c-web-application](aad/azure-spring-boot-starter-active-directory-b2c/aad-b2c-web-application)                             |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-resource-server-by-filter-stateless](aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter-stateless) |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-resource-server-by-filter](aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter)                     |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-resource-server-obo](aad/azure-spring-boot-starter-active-directory/aad-resource-server-obo)                                 |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-resource-server](aad/azure-spring-boot-starter-active-directory/aad-resource-server)                                         |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-web-application](aad/azure-spring-boot-starter-active-directory/aad-web-application)                                         |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-webapp-resource-server](aad/azure-spring-boot-starter-active-directory/aad-web-application-and-resource-server)              |
+| appconfiguration | [azure-spring-boot-starter-cosmos:3.8.0]                                                       | [azure-appconfiguration-conversion-sample-initial](appconfiguration/azure-appconfiguration-conversion-sample-initial)             |
```

Comment:
Is this a mistake?

Reply:
actually,azure-appconfiguration-conversion-sample-initial depends on azure-spring-boot-starter-cosmos
![image](https://user-images.githubusercontent.com/4465723/131937577-ea9c1ccf-f568-4b87-b9f4-ee170c67adcc.png)

Reply:
The folder/artifact name is confusing.